resources/public/api/conf/2.0/examples/foreign_property_period_summary_amend/consolidated_request.json

```diff
@@ -1,29 +1,33 @@
 {
   "foreignFhlEea": {
     "income": {
-      "rentAmount": 643.59
+      "rentAmount": 1123.89
     },
-    "expenditure": {
-      "consolidatedExpenses": 456.98
+    "expenses": {
```

Comment:
This is supposed to be a consolidated request. cant this be changed back to a consolidated example